neptune.yaml

```diff
@@ -86,5 +86,14 @@ parameters:
   erode_selem_size: 5
   dilate_selem_size: 5
 
+  #CRF
+  apply_crf: False
```

Comment:
@taraspiotr unfortunately right now it might not parse correctly. I always go with 1/0 to make sure it is passed correctly later.